Update to OIE overview page for PATEAM (#4780)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/oie-upgrade-overview/main/index.md

As the developer, identify the integrations that have been applied to the following use cases. Make sure that you test the end-to-end flow before and after the upgrade.

* Account creation & activation (including self-service registration)
* User authentication and user sign-in flows
* Password recovery
* Session management
  > **Caution:** Custom integrations using the Sessions API use `v1/sessions/{sessionid}` for back-channel session management.

### Helpful upgrade links

* [Prepare your customizations for upgrade](https://help.okta.com/okta_help.htm?type=oie&id=ext-custom-sign-in-page).
* [Identify how Okta is implemented](https://www.youtube.com/watch?v=gUqZUSeL_oM&list=PLIid085fSVduvUaN-gBdN1cudndqR9IH8&index=3).
* [Review the comprehensive list of Identity Engine feature changes](https://help.okta.com/okta_help.htm?type=oie&id=ext-oie-features) to identify any potential impact to your upgrade process.

## Org upgrade process

> Access the **OIE Upgrade Hub** in the Admin Console and review any outstanding tasks. If there are any remediation tasks required, those remediation guides appear in the **OIE Upgrade Hub**. If there are no remediations required, then your upgrade is more simple.

Reviewer: ```suggestion
Access the **OIE Upgrade Hub** in the Admin Console and review any outstanding tasks. If there are any remediation tasks required, those remediation guides appear in the **OIE Upgrade Hub**. If there are no remediations required, then your upgrade is easier.
```

For detailed org upgrade instructions, see the [Self-service upgrade process](https://help.okta.com/okta_help.htm?type=oie&id=ext-post-upgrade-validation-tests)

> Keep in mind the following considerations:

Reviewer: ```suggestion
Consider the following:
```

* After upgrading to Okta Identity Engine, your auth experience may continue to mimic Classic Engine. However, your admin experience changes and all Okta objects from Classic Engine are now compatible with Identity Engine.
* After you upgrade, the existing Okta-hosted Sign-In Widget works as-is.

> The rest of the upgrade process is defined in the next section. Which steps that you take are related to your deployment model. Okta has carefully considered how you can break up the upgrade steps to ensure that you maintain the best user experience across your applications.

Reviewer: ```suggestion
The rest of the upgrade process is defined in the next section. The steps that you take are related to your deployment model. Okta has carefully considered how you can break up the upgrade steps to ensure that you maintain the best user experience across your applications.
```

Reviewer: don't need which and that in the same sentence

    * [Upgrade your Okta Sign-In Widget](/docs/guides/oie-upgrade-sign-in-widget/) to the latest version.

    * Test your app to make sure the widget works.

    * Make any necessary [updates to the Sign-In Widget styling](/docs/guides/oie-upgrade-sign-in-widget-styling/) and [i18n properties](/docs/guides/oie-upgrade-sign-in-widget-i18n/).

    * Check for [deprecated functions or customizations](/docs/guides/oie-upgrade-sign-in-widget/main/#changes-to-widget-configuration-for-identity-engine) that may not be compatible with Identity Engine.

    **Are you embedding your authentication with an Okta SDK?**

    * Upgrade to the latest version of the Identity Engine SDK. See [Upgrade your app to the Identity Engine SDK](/docs/guides/oie-upgrade-api-sdk-to-oie-sdk/nodejs/main/) for detailed steps by language.

    * Add the appropriate Identity Engine SDK to your application code. See [Add the latest Auth SDKs to your applications](/docs/guides/oie-upgrade-add-sdk-to-your-app/nodejs/main/) for detailed steps by language.

3. Test all of your user experiences.

    * **Authentication:** Make sure that your users can sign in and sign out for the workflows that you support.

>     * **Self-service & Admin-initiated Password Recovery:** Make sure that users can recover their factors with no impact to the user experience.

Reviewer: ```suggestion
    * **Self-service and admin-initiated password recovery:** Make sure that users can recover their factors with no impact to the user experience.
```

>     * **New Account Activation:** Make sure that the account onboarding experience works as expected end-to-end to ensure that users have a smooth start.

Reviewer: ```suggestion
    * **New account activation:** Make sure that the account onboarding experience works as expected end-to-end to ensure that users have a smooth start.
```

    * **Self-service registration:** Make sure that users can sign up (including Factor enrollment if you support that) and that there are no interruptions during that process.

>     * **Session Management:** If you’re using the Sessions API, some [methods for the Sessions API aren't supported in Identity Engine](https://support.okta.com/help/s/article/v1sessionssessionid-API?language=en_US).

Reviewer: ```suggestion
    * **Session management:** If you’re using the Sessions API, some [methods for the Sessions API aren't supported in Identity Engine](https://support.okta.com/help/s/article/v1sessionssessionid-API?language=en_US).
```

Reviewer: I changed each of these to sentence case but if they're actual feature names I'm fine changing them back to capital.
